packages/libs/error-stack/tests/snapshots/test_debug__full__complex.snap

```diff
@@ -0,0 +1,52 @@
+---
+source: tests/test_debug.rs
+expression: "format!(\"{report:?}\")"
+---
+Context A
+├╴tests/test_debug.rs:452:14
+├╴Printable A
+│
+╰┬▶ Root error
+ │  ├╴tests/common.rs:147:5
+ │  ├╴Printable A
+ │  ├╴spantrace with 2 frames (1)
+ │  ╰╴backtrace with [n] frames (1)
+ │
+ ├▶ Root error
+ │  ├╴tests/common.rs:147:5
+ │  ├╴Printable B
+ │  ├╴Test
+ │  ├╴spantrace with 2 frames (2)
+ │  ├╴backtrace with [n] frames (2)
+ │  ╰╴1 additional opaque attachment
+ │
+ ╰▶ Root error
+    ├╴tests/common.rs:147:5
+    ├╴Printable B
+    ├╴Test
+    ├╴spantrace with 2 frames (3)
+    ├╴backtrace with [n] frames (3)
+    ╰╴3 additional opaque attachments
+
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+Backtrace No. 1
+  [redacted]
+
+
+Span Trace No. 1
+  [redacted]
```

Comment:
Were you able to determine why there are two line breaks here but only one linebreak between different attachments?

Reply:
I forgot to look into it, but I just figured out that during `Display` of backtrace, it emits one additional `\n`, which we do not anticipate.

Reply:
Ah, that could be the reason. Thanks for looking at that!

Reply:
fixed in [4b33137](https://github.com/hashintel/hash/pull/794/commits/4b331376c5eca6aeea2b498b5a68f924301d139f)

Reply:
Awesome!